src/pages/index.js

```diff
@@ -1,12 +1,27 @@
 import * as React from "react";
 import { DefaultLayout } from "../components/DefaultLayout";
+import { Button } from "../components/Button";
+import { Container } from "../components/Container";
 
 const IndexPage = () => {
   return (
-    <main className="container m-auto">
+    <Container>
       <title>Portfolio page</title>
-      <DefaultLayout></DefaultLayout>
-    </main>
+      <DefaultLayout>
+        <div className="grid grid-cols-1 lg:grid-cols-2 gap-x-32 my-56">
```

Comment:
I think `grid-cols-1` is the default value (by default `grid` has no columns). Try removing this class and see if it does any difference.

Reply:
Yes. You're right!